### ISSUE TITLE
Re-opening Train tab should show all metrics

### ISSUE THREAD

**Issue by torronen** [dotnet/machinelearning-modelbuilder#1582]
.mbconfig Train tab should keep all statistics for best model.
Right now only one of the metrics is shown, which in some cases is not relevant, at least until https://github.com/dotnet/machinelearning-modelbuilder/issues/1537 is possible

Workaround is to save the Model builder output file every time, so we can see abit more information from there

![image](https://user-images.githubusercontent.com/26261427/125162834-c1554d80-e192-11eb-8a2d-b26b1afc79b4.png)

**Comment by torronen** [comment]
This page could also show all of the data from .mbconfig training history in a table

**Comment by zewditu** [comment]
@briacht  for visibility.

**Comment by briacht** [comment]
@zewditu @LittleLittleCloud do all the evaluation metrics get logged in .mbconfig file?

**Comment by beccamc** [comment]
@torronen You want these metrics to be available? 

![image.png](https://images.zenhubusercontent.com/5d55c3d68fac840001314eaa/35b13960-38ca-4d38-939c-457d3030cda6)

@briacht No, the MBConfig saves score and train time. 

![image.png](https://images.zenhubusercontent.com/5d55c3d68fac840001314eaa/8cedbabd-53d3-4193-b13c-ed687b6f5ef7)

**Comment by briacht** [comment]
@torronen If mbconfig json file logged these metrics, would that fill your requirement?

**Comment by torronen** [comment]
@briacht @beccamc yes, it would be convenient if shown on the summary page UI. Currently, I just save the output (in beccamc's screenshot) as a text file which is good enough. 

The reason is to check how the different datasets performed, for example, I have one model predict "WillRain" and another model predict "RainAmount". After 1 month I may receive new data (for example, new preprocessing), then I want to check which is probably faster to train and might give better accuracy.

**Comment by JakeRadMSFT** [comment]
Related to #1591

**Comment by beccamc** [comment]
This needs more thought on where we want to save these metrics and how we want to display them in UI. MBconfig file? AutoML run history file?

**Comment by LittleLittleCloud** [comment]
I'd vote for saving metric name in mbconfig, under a sub-root
```json
{
....
     "advanceTrainingOption":{
        ... other options
        "metricName": "RMSE"
        ....
    }
}
```

**Comment by beccamc** [comment]
Similar request here https://github.com/dotnet/machinelearning-modelbuilder/issues/1056:

> VS Extension: Save the content of the output window (with the logs about the training) as a text file when adding the projects *ML.Model, *ML.ConsoleApp
